generator/core/tsl_generator.py

```diff
@@ -111,6 +112,64 @@ def generate(self, relevant_hardware_flags: List[str] = None):
             extensions_dict = {"generated_extensions": extensions_list}
             yaml_store(config.tsl_extensions_yaml_output_path, extensions_dict)
         relevant_primitives_class_set: TSLPrimitiveClassSet = slicer.slice_primitives(self.__tsl_primitiveclass_set)
+        
+        """ Only filter if any relevant primitves are set. """
+        if config.get_config_entry("relevant_primitives"):
```

Comment:
Please check for list containing only empty strings

Reply:
done as filtering based on string length in the config parameter.